infra/es_cluster.tf

```diff
@@ -188,13 +209,17 @@ RUN bin/elasticsearch-plugin install --batch discovery-gce
 COPY es.yml /usr/share/elasticsearch/config/elasticsearch.yml
 EOF
 
+mkdir -p /root/es-data
+chown 1000:0 /root/es-data
+
 docker build -t es .
 docker run -d \
            --restart on-failure \
            --name es \
            -p 9200:9200 \
            -p 9300:9300 \
            -e ES_JAVA_OPTS="-Xmx${local.es_clusters[count.index].xmx} -Xms${local.es_clusters[count.index].xmx}" \
+           -v /root/es-data:/usr/share/elasticsearch/data \
```

Comment:
This explicitly saves the data on the host, so we don't lose it if the container crashes or gets deleted. (We still lose it if the machine itself gets wiped, though.)